docs/development_suite/identity-and-access-management/service-account-management.md

```diff
@@ -0,0 +1,82 @@
+
+:::note
+To find out more about Roles check out the available [Capabilities](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#users-capabilities-inside-console) that can be assigned to an identity.
+:::
+
+### What can you do with a Service Account?
+
+Virtually anything a User can do inside the Console can be entrusted to a dedicated Service Account, which will perform the desired actions automatically.
+
+For instance, a Service Account could be granted permissions to make automatic deploys, which can greatly streamline the process of deploying code changes to an application. The Service Account could use its credentials to access the appropriate deployment resources and automatically deploy new versions of the application as soon as new code is pushed to the source code repository.  
+This allows teams to implement a CI/CD workflow that can improve the speed, reliability, and scalability of their software development processes. 
+
+A Service Account can also be used to monitor and log Kubernetes resources, such as pods, deployments and services. Once authorized, the Service Account can use logging and monitoring tools to collect and analyze data on resource usage, performance metrics, and other important indicators.
+
+## Managing Company Service Accounts
+
+A User with enough administrative permission on a specific Company will be able to view the existing Service Accounts in the Company, to add new ones and to change the existing Service Accounts' Roles.
+
+<!-- TODO: inserire screenshot dell'admin portal per la Company, con filtro attivo per identity type = Service Account -->
+
+:::caution
+Please note that some permissions defined by the Company Role may be inherited on the Projects and Runtime Environment owned by the Company itself.  
+Always pay attention when assigning Roles in order to avoid providing undesired access to resources!
+:::
+
+### Adding a new Service Account
+
+The Company Owner can add a new Service Account by clicking the *Add Service Account* option inside the *Add User* split button, on the top-right corner of the Identities page. 
+
+<!-- TODO: SCREENSHOT OF COMPANY ADD SA MODAL -->
+
+The the Service Account creation process will require the following information:
+- Name: a human-readable name to identify the Service Account
+- Role: the Company Role to be assigned to the Service Account
+- Authentication method: it is necessary to verify the identity of the Service Account and can be of two types <!-- TODO: SCREENSHOT OF THE DIFFERENT FLOWS -->
+    - `client_secret_basic`: the Service Account authenticates by presenting its `client_id` and `client_secret` in the Authorization header of the request, in the format `Authorization: Basic <base64 encoded client_id:client_secret>`. The server then decodes the header and validates the credentials against its records to authenticate the client.  
```

Comment:
we can link here the client credentials docs for the two flows

https://docs.mia-platform.eu/docs/runtime_suite/client-credentials/usage#client-secret-basic

https://docs.mia-platform.eu/docs/runtime_suite/client-credentials/usage#private-key-jwt

Reply:
```suggestion
    - `client_secret_basic`: the Service Account authenticates by presenting its `client_id` and `client_secret` in the Authorization header of the request, in the format `Authorization: Basic <base64 encoded client_id:client_secret>`. The Console then decodes the header and validates the credentials against its records to authenticate the client.  
```